### PR TITLE
SER-39 Generate/Provide appropriate headers to pandora services

### PR DESCRIPTION
**Pull request by pchojnacki**
Added generating following headers to forwarded requests:
- 'X-Beacon-Id'
- 'X-Client-Ip'
- 'X-Trace-Id'
- 'Fastly-Client-IP'

Additionally I fixed a bug with returning empty response when service binding was not found.

Also following response headers has been added:
- X-Upstream-Name
- X-Trace-Id
- X-Upstream-Name

### CODE REVIEW

**Comment by sqreek** [comment]
LGTM

**Comment by pchojnacki** [comment]
Ping @sqreek

File: t/basic.t [hunk 1]
```diff
@@ -8,17 +8,34 @@ use lib dirname(abs_path($0));
...
+    
+    location /info {
+        content_by_lua_block {            
+            ngx.say('{"status": "git", "user_id": "90061"}');
+        }
+    }    
+CONFIG
 
-plan tests => repeat_each(1) * (2 * blocks());
+plan tests => repeat_each() * (3 * blocks()) + 1; # i don't understand this :(
```
Comment (sqreek) [inline t/basic.t]: Me either

File: t/basic.t [hunk 2]
```diff
@@ -27,52 +44,74 @@ __DATA__
     location /echo {
         echo_before_body hello;
         echo world;
-    }
+    }
```
Comment (sqreek) [inline t/basic.t]: tab?

File: t/basic.t [hunk 3]
```diff
@@ -27,52 +44,74 @@ __DATA__
...
+"Fastly-Client-IP: 10.10.10.10
+Cookie: wikia_beacon_id=somebacon
+X-User-Id: someUserId
+X-Wikia-UserId: someUserId"
+--- request
+    GET /test/headers
+--- response_headers
+Fastly-Client-IP: 10.10.10.10
+X-Client-Ip: 10.10.10.10
+X-Beacon-Id: somebacon
```
Comment (sqreek) [inline t/basic.t]: some bacon :)

File: README.md [hunk 4]
```diff
@@ -52,10 +52,10 @@ The integration tests require the following environment variables:
 If the `access_token` cookie exists and is not empty then an attempt will be
 made to authenticate the token against helios and retrieve the user id of the
 user to whom the token belongs. If a user id is returned from helios then that
-user id will be provided in the `X-Wikia-UserId` header sent to the service.
+user id will be provided in the `X-User-Id` header sent to the service.
```
Comment (sqreek) [inline README.md]: Add information that we will forward both, `X-User-Id` and `X-Wikia-User-Id`.

We need deprecate `X-Wikia-User-Id` in services first.

**Comment by sqreek** [comment]
LGTM

**Pull request merged**